### ISSUE TITLE
:version specifiers for plugins are not checked

### ISSUE THREAD

**Issue by karenetheridge** [rjbs/Pod-Weaver#38]
The `:version` checks, that Dist::Zilla performs for plugin configurations, does not appear to be done here, either from parsing `weaver.ini` nor from configurations passed directly from the `configure` sub in a plugin bundle.

Dist::Zilla is doing this in Dist::Zilla::Role::PluginBundle::Easy with an earlier invocation of `load_class` with the specified version; Pod::Weaver could do the same.  But could this perhaps be generalized back to Config::MVP itself?

**Comment by rjbs** [comment]
Yeah, MVP should have a means to do this in the assembler, I think.
